main.go

```diff
@@ -245,10 +226,11 @@ func main() {
 		}
 
 		ctx.JSON(http.StatusOK, map[string]interface{}{
-			"address":     wallet.EthereumAddress,
-			"turnkeyUuid": wallet.TurnkeyUUID,
-			"balance":     formatBalance(balance),
-			"dropsLeft":   wallet.DropsLeft(),
+			"address":               wallet.EthereumAddress,
+			"turnkeyUuid":           wallet.TurnkeyUUID,
+			"balance":               formatBalance(balance),
+			"dropsLeft":             wallet.DropsLeft(),
+			"turnkeyOrganizationId": user.SubOrganizationId.String,
```

Comment:
I don't have anything against returning this here (it's easy, doesn't hurt anything), but I wonder why you had to do this: is the org ID not part of the broader auth context? If we have more complex flows later it might be worth consolidating so there's one context that's user/org aware.